app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java

```diff
@@ -137,11 +138,17 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                                     registerPass.setError("The password is invalid it must 6 characters at least");
                                     registerPass.requestFocus();
                                     break;
+
                             }
                         }
                     }
+
+
```

Comment:
Remove extra lines everywhere. A small piece of advice, while committing, just see the difference in the changed files and remove which should not be there.